internal/policies/mount/mount_test.go

```diff
@@ -63,33 +79,61 @@ func TestApplyPolicy(t *testing.T) {
-		"user, successfully apply policy filtering out unsupported keys":  {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "user-mounts"}},
+		"user, successfully apply policy for entry with one value":       {},
+		"user, successfully apply policy for entry with multiple values": {entries: []string{"entry with multiple values"}},
+		"user, successfully apply policy for entry with repeated values": {entries: []string{"entry with repeated values"}},
+		"user, successfully apply policy filtering out unsupported keys": {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "user-mounts"}},
 
 		// Special cases.
-		"successfully apply policy with kerberos auth tags": {entries: []string{"entry with kerberos auth tags"}},
+		"user, successfully apply policy with kerberos auth tags": {entries: []string{"entry with kerberos auth tags"}},
 
 		// Badly formatted entries.
 		"user, successfully apply policy trimming whitespaces":           {entries: []string{"entry with spaces"}},
 		"user, successfully apply policy trimming sequential linebreaks": {entries: []string{"entry with multiple linebreaks"}},
-		"user, creates only users_user dir if the entry is empty":        {entries: []string{"entry with no value"}},
-		"user, creates only users dir if there are no entries":           {entries: []string{"no entries"}},
+		"user, creates only dirs if the entry is empty":                  {entries: []string{"entry with no value"}},
+		"user, creates only dirs if there are no entries":                {entries: []string{"no entries"}},
 
 		// Policy refresh.
 		"user, mount file is removed on refreshing policy with no entries":                    {secondCall: []string{"no entries"}},
 		"user, mount file is removed on refreshing policy with an empty entry":                {secondCall: []string{"entry with no value"}},
 		"user, mount file is updated on refreshing policy with an entry with multiple values": {secondCall: []string{"entry with multiple values"}},
 
+		/**************************** SYSTEM ***************************/
+		// Success cases.
+		"system, successfully apply policy for entry with one value":       {isComputer: true},
+		"system, successfully apply policy for entry with multiple values": {entries: []string{"entry with multiple values"}, isComputer: true},
+		"system, successfully apply policy for entry with repeated values": {entries: []string{"entry with repeated values"}, isComputer: true},
+		"system, successfully apply policy filtering out unsupported keys": {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "system-mounts"}, isComputer: true},
+
+		// Special cases.
+		"system, successfully apply policy with kerberos tagged values": {entries: []string{"entry with kerberos auth tags"}, isComputer: true},
```

Comment:
I think we should have a duplicated entry section in the tests, for the 2 use cases I mentioned in https://github.com/ubuntu/adsys/pull/522#discussion_r1040856553:

```
We took that offline yesterday FTR. The TLDR is:

* let’s have the first entry winning
* print warnings in case of duplication with different parameters (today [krb5]/no tag, later mount options?)
```

Reply:
Keeping it opened following the discussion on MM (for the code itself which should warn instead of debug and be more precise for the admin to understand what’s happening)